Global: color v3 global oiio transcoder plugin (#4291)
ynput/OpenPype: website/docs/project_settings/settings_project_global.md

The **colorspace name** value is a raw string input and no validation is run after saving project settings. We recommend to open the specified `config.ocio` file and copy pasting the exact colorspace names.
:::

### Extract OIIO Transcode

> There is profile configurable (see lower) plugin which allows to transcode any incoming representation to one or multiple new representations (configured in `Output Definitions`) with different target colorspaces.

Reviewer: ```suggestion
There is profile configurable plugin which allows to transcode any incoming representation to one or multiple new representations (configured in `Output Definitions`) with different target colorspaces.
```

Dont think we need to point people just below.

> There is profile configurable plugin which allows to transcode any incoming representation to one or multiple new representations (configured in `Output Definitions`) with different target colorspaces.
> Plugin expects instances with filled dictionary `colorspaceData` on a representation. This data contains information about source colorspace and must be collected for transcoding.
> `oiiotool` is used for transcoding, eg. `oiiotool` must be present in `vendor/bin/oiio` or environment variable `OPENPYPE_OIIO_PATHS` must be provided for custom oiio installation.
> 

Reviewer 2: ```suggestion
OIIOTools transcoder plugin with configurable output presets. Any incoming representation with `colorspaceData` is convertable to single or multiple representations with different target colorspaces or display and viewer names found in linked **config.ocio** file.

`oiiotool` is used for transcoding, eg. `oiiotool` must be present in `vendor/bin/oiio` or environment variable `OPENPYPE_OIIO_PATHS` must be provided for custom oiio installation.

```

Notable parameters:
- **`Delete Original Representation`** - keep or remove original representation. If old representation is kept, but there is new transcoded representation with 'Create review' tag, original representation looses its 'review' tag if present.

> - **`Extension`** - target extension, could be empty - original extension is used

Reviewer: ```suggestion
- **`Extension`** - target extension. If left empty, original extension is used.
```

> - **`Colorspace`** - target colorspace - must be available in used color config

Reviewer: ```suggestion
- **`Colorspace`** - target colorspace, which must be available in used color config.
```

> - **`Display & View`** - transcoding into colorspace OR into display and viewer space could be used. (It is disjunctive: Colorspace & nothing in Display and View or opposite)

Reviewer: ```suggestion
- **`Display & View`** - transcoding into colorspace or into display and viewer space could be used. Cannot use both `Colorspace` and `Display & View` at the same time.
```

> - **`Arguments`** - special additional command line arguments for `oiiotool`

Reviewer: ```suggestion
- **`Arguments`** - special additional command line arguments for `oiiotool`.
```

Example here describes use case for creation of new color coded review of png image sequence. Original representation's files are kept intact, review is created from transcoded files, but these files are removed in cleanup process.

> ![global_oiio_transcode](assets/global_oiio_transcode.png)

Reviewer: Could we have another example here for transcoding just the review for a Maya beauty render?

Reviewer: Think the main things would be:

- no `Colorspace` or `Display & View` cause that is taken from the DCC (Maya).
- subset regex to only transcode "beauty" render layer.

Author: How this should work? I don't have `Colorspace` collected from DCC

The **colorspace name** value is a raw string input and no validation is run after saving project settings. We recommend to open the specified `config.ocio` file and copy pasting the exact colorspace names.
:::

### Extract OIIO Transcode
OIIOTools transcoder plugin with configurable output presets. Any incoming representation with `colorspaceData` is convertable to single or multiple representations with different target colorspaces or display and viewer names found in linked **config.ocio** file.

`oiiotool` is used for transcoding, eg. `oiiotool` must be present in `vendor/bin/oiio` or environment variable `OPENPYPE_OIIO_PATHS` must be provided for custom oiio installation.

Notable parameters:

> - **`Delete Original Representation`** - keep or remove original representation. If old representation is kept, but there is new transcoded representation with 'Create review' tag, original representation looses its 'review' tag if present.

Reviewer 3: ```suggestion
- **`Delete Original Representation`** - keep or remove original representation. If old representation is kept, but there is new transcoded representation with 'Create review' tag, original representation loses its 'review' tag if present.
```
